simulation/__main__.py

```diff
@@ -143,8 +144,10 @@ def check_string_is_isoformat_datetime(string) -> str:
         args.gas_oil_boiler_ban_date,
     )
 
-    if args.history_file.startswith("gs://"):
-        history = list(history)
+    history = list(history)
+
+    with open(args.history_file + ".pkl", "wb") as file:
```

Comment:
`args.history_file = output.jsonl` so `args.history_file + ".pkl" = output.jsonl.pkl`. I know this will work, but a confusing naming convention - did you mean:
```suggestion
    with open(args.history_file.replace("jsonl", "pkl") as file
```
or do you want to leave it as-is?

Note: we don't actually check that `args.history_file` ends in `jsonl` anywhere, and it clearly should do in order for the output to save/load properly.

Reply:
No, it was intentional. It's only for debugging.

>Note: we don't actually check that args.history_file ends in jsonl anywhere, and it clearly should do in order for the output to save/load properly.

It doesn't actually - the extension doesn't matter.